src/crimson/net/SocketConnection.cc

```diff
@@ -555,7 +560,10 @@ SocketConnection::handle_keepalive2_ack()
     .then([this] (auto buf) {
       auto t = reinterpret_cast<const ceph_timespec*>(buf.get());
       k.ack_stamp = *t;
-      std::cout << "keepalive2 ack " << t->tv_sec << std::endl;
+      std::ostringstream out;
+      logger().info("{} keepalive2 ack {}",
+                    _conn_prefix(out).str(),
```

Comment:
if we offer `<<` operator, there is no need to create an `ostringstream` here, as `fmt::format_arg()` is able to take care of the printed value which offers `operator <<()`.

Reply:
fixed.